download_geo_matrix.py

```diff
@@ -27,20 +27,21 @@ def main(argv):
     args = parser.parse_args(argv)
 
     accessions = args.accessions.split(',')
+    namespace_uuid = uuid.uuid4()
```

Comment:
The namespace uuid needs to be a defined constant (not randomly generated each run) so that given an accession id we would regenerate the same project uuid.
```suggestion
    namespace_uuid = uuid.UUID('296e1002-1c99-4877-bb7f-bb6a3b752638')
```